docs/UserGuide.md

```diff
@@ -168,22 +177,26 @@ The `Module tab` includes `Lessons`, `Tasks` and `Contacts` that are related to
  
 1. `Lessons`: List of **lessons** for this module.
 
-2. `Tasks`: List of **tasks** in **chronological** order. *Tips: Don't leave tutorials or assignments until the last day, chances are you not gonna finish it!*
+2. `Tasks`: List of **tasks** in **chronological** order. 
+  :bulb: *Don't leave tutorials or assignments until the last day, chances are you not gonna finish it!*
```

Comment:
Don't leave tutorials or assignments until the last day, chances are you won't finish it in time!